package-overrides/github/a5hik/ng-sortable@1.1.9.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "css": "*"
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "main": "ng-sortable",
+  "registry": "jspm",
+  "shim": {
+    "ng-sortable": {
+      "deps": [
+        "./ng-sortable.min.css!"
```

Comment:
You can set the main to a CSS file, but in this case we want the js and CSS together. I am considering allowing virtual modules for this to allow variations of with and without CSS included, but for now what you've done is exactly the best advised way of going about it.